eos/rare-b-decays/nonlocal-formfactors.hh

```diff
@@ -87,6 +87,13 @@ namespace eos
 
             ///@}
 
+            ///@name Evaluate the ratio between non-local and local formfactors.
+            ///@{
+
+            virtual complex<double> normalized_plus(const double & /*q2*/) const = 0;
```

Comment:
It's OK to have a named argument here. Warnings only appear in the definition if ``q2`` is unused.